packages/eslint-plugin-react/index.js

```diff
@@ -18,6 +18,9 @@ module.exports = {
         // prefer "<></>" syntax over <React.Fragment></React.Fragment>
         'react/jsx-fragments': ['error', 'syntax'],
 
+        'react-hooks/rules-of-hooks': 'error',
+        'react-hooks/exhaustive-deps': 'warn',
```

Comment:
Can we use 'error' instead? We dont pay attention at lint warnings 😕

Reply:
In some case @flepretre showed me that we don't want to trigger effect on some "dependencies" props. Or in order to make a `didMount` like usage.

Not sure of what to do

Reply:
Ok, let it as a warning 😕